tests/utils.py

```diff
@@ -46,7 +46,7 @@ def assert_session_model_error(
     location: Location = Location((1, 1)),
 ) -> None:
     with pytest.raises(RecordFluxError, match=regex):
-        Session(
+        UnprovenSession(
```

Comment:
`Session` could be used directly here (as before).

Reply:
Reverted in d73322c16d65cf8838334a7916105f42e4e0f9ec.